_docs/resources/engaging-helix.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Engaging Helix
+layout: docs
+status: stable
+resource: true
+category: Resources
+last-modified: 2018-07-11
+usage: >
+   The long and short of it is that you should reach out the the Helix team anytime the team might be helpful to you, but it’s not always easy to know what that is. Use the following scenarios as a guide for when you should contact the Helix team.
+---
+
```

Comment:
Maybe this is where we could include a short and sweet About section? Maybe at the end of the About section you can list the ways in which I can engage with the Helix team? Like, maybe I already know I have a legit reason to engage, so how do I do that? If someone still has questions about when to engage, they can read the awesome cartoons.